### DISCUSSION TITLE
Import to InfluxDB - Add Tutorial?

### DISCUSSION THREAD

**Discussion by mgorfer** [tcgoetz/GarminDB#190] [Ideas]
Because I wanted to have more control over how to plot my metrics, I imported the data from the local SQLite DBs into my InfluxDB. The process was surprisingly straight-forward, I had just to write a simple config script for each separate SQLite DB file, which are supported natively in InfluxDB, and now can access all my metrics via Grafana. My data exists from 2019, and it is really nice to have it interactively in a Dashboard there.

If you think that this would be a plus for this project, I would gladly share my InfluxDB config files, my Grafana Dashboard, and a small tutorial how to set up all of this. Still took me a while to figure out everything.

I used docker-compose to set up InfluxDB and Grafana.
Ideally, also GarminDB should be available as a Docker Container, then this would be nearly a one click setup process.
I already found an unofficial Container, https://github.com/8cH9azbsFifZ/docker-garmindb, but this is not ARM64 compatible, unfortunately. 
I just made my own ARM64 docker container then, and now I run everything with one docker-compose file. But I am not sure what would be the easiest approach for the average user of GarminDB.

**Comment by tcgoetz** [comment]
I think that will be useful to others. I suggest you commit it all to a GitHub repository of your own and then we can link to it from this page:  https://github.com/tcgoetz/GarminDB/wiki/Related-Projects

**Comment by tcgoetz** [comment]
Also, if you set the dependencies correctly on your repo, it will show up here: https://github.com/tcgoetz/GarminDB/network/dependents

**Comment by prostolyubo** [comment]
Hi! @mgorfer that's exactly what I did. I made my own container with GarminDB, shared the volumes with grafana and realized that building queries for SQLite in Grafana is a nightmare and I'm looking for a way to switch to a wider supported database. 
I've seen the "db : type" field in json config but it seems only `sqlite` and `mysql` are supported.

**Comment by arpanghosh8453** [comment]
@mgorfer I am the developer of [Fitbit Fetch](https://github.com/arpanghosh8453/public-fitbit-projects) and would like to see the config and how you did this integration for Garmin devices. There seems to be a lot of interest around this : https://www.reddit.com/r/Garmin/comments/1jset6s/will_you_be_interested_in_a_web_dashboard_like

I can design the one click setup and containerize this so it can be used by many other people. Please let me know and thank you.

**Comment by tcgoetz** [comment]
I think you should create your own repo for your config files and build and then we can add them to the [related projects](https://github.com/tcgoetz/GarminDB/wiki/Related-Projects) section.You can also mark your repo as dependent on GarminDB
